ngraph/core/src/distributed.cpp

```diff
@@ -12,7 +12,7 @@ using namespace ngraph;
 
 namespace ov {
 template <>
-EnumNames<ngraph::reduction::Type>& EnumNames<ngraph::reduction::Type>::get() {
+NGRAPH_API EnumNames<ngraph::reduction::Type>& EnumNames<ngraph::reduction::Type>::get() {
```

Comment:
Can we use OPENVINO_API?

Reply:
I used `NGRAPH_API` for consistently with other places